scripts/vendor.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+"""
+Fetch and bundles the hcloud package inside the collection.
+
+Fetch the desired version `HCLOUD_VERSION` from https://github.com/hetznercloud/hcloud-python
+`HCLOUD_SOURCE_URL` using git, apply some code modifications to comply with ansible,
+move the modified files at the vendor location `HCLOUD_VENDOR_PATH`.
+"""
+
+import logging
+import re
+from pathlib import Path
+from shutil import move, rmtree
+from subprocess import check_call
+from tempfile import TemporaryDirectory
+from textwrap import dedent
+
+logger = logging.getLogger("vendor")
+
+HCLOUD_SOURCE_URL = "https://github.com/hetznercloud/hcloud-python"
+HCLOUD_VERSION = "v1.24.0"
+HCLOUD_VENDOR_PATH = "plugins/module_utils/vendor/hcloud"
+
+
+def apply_code_modifications(source_path: Path):
+    # __version___.py is considered as an invalid filename in module_utils/
```

Comment:
I would explicitly mention that this is a problem for Galaxy importer (since it isn't a problem for any other tool I know, including ansible-core)
```suggestion
    # __version___.py is considered as an invalid filename in module_utils/ by Galaxy importer
```

Reply:
Alright, I wanted to report this, but the ansible-galaxy-importer repository doesn't have a GitHub bug tracker and I didn't check if I can register to the Redhat bug tracker. Is that ok if I leave this to you ?

Reply:
I don't have a RH account, so I won't be able to register anything on the RH bug tracker.